README.md

```diff
@@ -13,34 +13,37 @@ On your project root run:
 
 Add the following maven repository to your project's `build.gradle` file:
 
-  ```java
-  allprojects {
-    repositories {
-      maven {
-        url 'https://s3.amazonaws.com/salesforcesos.com/android/maven/release'
-      }
+```java
+allprojects {
+  repositories {
+    maven {
+      url 'https://s3.amazonaws.com/salesforcesos.com/android/maven/release'
     }
   }
-  ```
+}
+```
+
+Add the following pod sources to your project's `Podfile` file:
+
+```js
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/goinstant/pods-specs-public'
+```
```

Comment:
I didn't add this in the last version release as I didn't think it would be necessary to have these pod sources declared. I added and then removed this in our project's Podfile (to test whether it was necessary or not), but when clearing the pod cache, if these pod sources are not in the project's Podfile that is using the lib, it won't be possible to install the Salesforce SDK.